pandas/tseries/tools.py

```diff
@@ -238,6 +238,18 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
         datetime strings, and if it can be inferred, switch to a faster
         method of parsing them. In some cases this can increase the parsing
         speed by ~5-10x.
+    origin : scalar convertible to Timestamp / string ('julian', 'epoch'),
+        default 'epoch'.
+        Define relative offset for the returned dates.
```

Comment:
"returned dates" -> isn't the offset to interpret the input? (as the returned dates should always be correct dates, not with an offset)

Also, can you specify this is only relevant when converting integers (based on the `unit` kwarg)?